app/assets/javascripts/user/camps_registration.js

```diff
@@ -0,0 +1,60 @@
+$(document).on('ready turbolinks:load', () => {
+
+  hideForms();
+  modalScript();
+
+  if ($('#diability-input-field').val() != "") {
+    $('#medical-services-question').addClass("d-none");
+    $('#disability-input').removeClass("d-none");
+  } else if ($('#diability-input-field').val() == "") {
+    $('#medical-services-question').removeClass("d-none");
+    $('#disability-input').addClass("d-none");
+
+    if ($('#medical-input').val() != "") {
```

Comment:
Lets try to prevent `!` and use `==` comparison first and this block's code in `else`.